easybuild/easyconfigs/h/HDF5/HDF5-1.10.1-foss-2020a.eb

```diff
@@ -0,0 +1,29 @@
+name = 'HDF5'
+version = '1.10.1'
+
+homepage = 'https://portal.hdfgroup.org/display/support'
+description = """HDF5 is a data model, library, and file format for storing and managing data.
+ It supports an unlimited variety of datatypes, and is designed for flexible
+ and efficient I/O and for high volume and complex data."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = [
+    'https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(version_major_minor)s/hdf5-%(version)s/src']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['%(name)s-%(version)s-mpi_deprecated_methods.patch']
+checksums = [
+    '048a9d149fb99aaa1680a712963f5a78e9c43b588d0e79d55e06760ec377c172',  # hdf5-1.10.1.tar.gz
+    # HDF5-1.10.1-mpi_deprecated_methods.patch
+    '3d564dca389eea5ce09e88476fa4c048ca0c79690859a133b09a46db58a60c88',
+]
+
+dependencies = [
+    ('OpenSSL', '1.1.1e'),
+    ('zlib', '1.2.11'),
+    ('Szip', '2.1.1'),
+    ('libxml2', '2.9.10'),
+]
+
```

Comment:
We should probably turn on the testing, i.e.
runtest = "test"
on all new HDF5 easyconfigs.
Depends on having https://github.com/easybuilders/easybuild-easyblocks/pull/2250 though.

@boegel thoughts on doing this?

Reply:
@akesandgren Please open a dedicated issue for that, and we can do it in the most recent HDF5 easyconfigs?